src/pages/upgrades/get-involved/bug-bounty.js

```diff
@@ -157,7 +157,7 @@ const Client = styled(GatsbyImage)`
 
 const ClientIntro = styled.p`
   text-transform: uppercase;
-  font-size: 14px;
+  font-size: 14rem;
```

Comment:
While this is an interesting design choice I don't think it's cohesive with what we have at the moment 🤪

<img width="1274" alt="Screenshot 2022-01-31 at 10 08 08" src="https://user-images.githubusercontent.com/62268199/151775439-9530d4a3-da53-4982-a654-14663109361e.png">


```suggestion
  font-size: 0.875rem;
```